lib/isReachable.js

```diff
@@ -0,0 +1,23 @@
+// A directed graph is represented as an array of 'edges'.
+// Each 'edge' is composed of a start and end node.
+
+/* Example:
+[
+	{from: 'A', to: 'C'},
+	{from: 'B', to: 'D'},
+	{from: 'C', to: 'E'},
+	{from: 'D', to: 'A'},
+	{from: 'E', to: 'F'}
+];
+*/
+
+// Given a directed graph, a character representing a start node,
+// and a character representing a destination, determine whether 
+// it is possible to reach that destination by beginning at the start 
+// and traversing the graph.
```

Comment:
~~I assume edges can be traveled in either direction. If so then maybe we should just describe the edges in the input so we don't imply a direction. something like `[ [ 'A', 'C' ], [ 'B', 'D' ] ]`? What do you think?~~

Nevermind looking at the tests the graph is not traversable in any direction. I'm guessing the name 'Directed Graph' would give that away. 🤦‍♂️